garage/tf/models/gaussian_mlp_model.py

```diff
@@ -5,7 +5,6 @@
 from garage.experiment import deterministic
 from garage.tf.core.mlp import mlp
 from garage.tf.core.parameter import parameter
-from garage.tf.distributions import DiagonalGaussian
```

Comment:
Why delete this?

Reply:
I thought we might put the distribution object in the policy. But right now I think we should put it in the model as gaussian mlp model will always have a distribution.

Reply:
Thank you for the explanation. Putting the distribution in model makes more sense